app/assets/stylesheets/govuk_publishing_components/lib/print_support.scss

```diff
@@ -0,0 +1,29 @@
+// A helper class for removing column layouts when printing.
+// This should be applied to row/column layouts, by adding the
+// class to elements with a `govuk-grid-row` class. The columns
+// will then print the at full width of the page.
+// Note that only the immediate child columns will be affected.
+// If a child column also contains a row/column layout of
+// further elements, these will be unaffected and will retain
+// their existing column layout. Use the class again on these
+// nested grids to apply fullwidth column printing if required.
+
+// stylelint-disable declaration-no-important
+@include govuk-media-query($media-type: print) {
+  .gem-print-columns-none {
+    width: 100%;
+
+    > .govuk-width-container {
+      margin: 0;
+      max-width: none;
+    }
+
+    > [class*="govuk-grid-column"] {
+      position: static !important;
+      float: none !important;
+      clear: both !important;
+      width: 100% !important;
```

Comment:
Just want to check, is `!important` needed for all of these?

Reply:
Without checking every possible layout/view I can't be sure, but the `position:static` needs to trump the `position:sticky` used in html publication on the contents list which currently has higher specificity. 

Given that, it seemed likely there might be other places where rules like this might exist in other layouts, so I felt using `!important` on these properties would ensure we're not caught out. I don't see the issue with using `!important` in these print styles, since they are only used for print, they're neatly encapsulated and they are at the bottom of our CSS specificity tree.

Reply:
Okay, but for reference I've tested this and none of the `!important`s are needed. Position static doesn't need to override position sticky, because sticky doesn't apply to print.

I think I'd rather do what we know is needed, then test this on other layouts and iterate it as we go if necessary.

Reply:
Personally I wouldn't rely on the emulated print view in dev tools. I appreciate it's easier to play around with style changes but it's usually way off what the actual print appearance is.

Reply:
That was my point :) This is more for others who may be using emulated views and wondering why the contents-list is overlaying the rest of the page. It's a useful fix, so why can't we include it?

Reply:
That's fine 👍